src/platform/testing/e2e/cypress/plugins/index.js

```diff
@@ -45,8 +45,8 @@ module.exports = on => {
 
       if (!initialized) {
         // Wipe existing fixtures and recreate tmp dir for a clean state.
-        fs.rmdirSync(TMP_FIXTURES_PATH, { recursive: true });
-        fs.mkdirSync(TMP_FIXTURES_PATH, { recursive: true });
+        fs.removeSync(TMP_FIXTURES_PATH);
+        fs.ensureDirSync(TMP_FIXTURES_PATH);
```

Comment:
Needed to use `fs-extra` since I'm making Cypress use our version of Node instead of the bundled version. But our version is quite old, so there are no recursive dir commands...